src/plugins/settings/setting.ts

```diff
@@ -18,7 +18,9 @@ export abstract class UIPSetting extends ESLBaseElement {
   /** [Target's]{@link target} attribute which is changed by setting. */
   @attr() public attribute: string;
   /** Target to which setting's changes are attached. */
-  @attr() public target: string;
+  public get target(): string {
+    return this.getAttribute('target') || this.$settings.target;
+  }
```

Comment:
Can't we change it from JS?
Why do we commit the setter part of the property for this attribute?